host/xtest/pkcs11_1000.c

```diff
@@ -4228,6 +4228,10 @@ static void xtest_pkcs11_test_1017(ADBG_Case_t *c)
 	if (!ADBG_EXPECT_CK_OK(c, rv))
 		goto out;
 
+	rv = C_EncryptInit(session, &cktest_aes_cbc_mechanism, aes_key_enc);
```

Comment:
Could you add an explicit inline comment like `/* Initializing again the encryption again should not terminate the operation */`

Reply:
OK